### DISCUSSION TITLE
更新青龙或者重装，数据丢失。

### DISCUSSION THREAD

**Discussion by blue-ice-layer** [whyour/qinglong#1830] [Q&A]
重装青龙或者使用watchtower更新后，跟新装的青龙一样，以前的库、环境变量和依赖都没了。有没有人有同样问题，求大佬解决一下。

**Comment by BenMake** [comment]
+1 同样出现了， 如何解决求助

**Reply by blue-ice-layer** [reply]
docker容器的话，把容器/ql/data目录挂载到你指定目录
